msteams-platform/bots/what-are-bots.md

```diff
@@ -46,17 +46,24 @@ You can use the Teams bot APIs to get information for members of a chat or team.
 <!--- TBD: For quick scanning, see if the above information can be itemized as a list.
 --->
 
-## Next step
+## Global endpoint for bots API
```

Comment:
Overall this needs more motivation-- what problem does this solve? what are the scenarios in which a developer would find this feature useful?

Reply:
When does a developer *not* need to worry about this, and can do what they were already doing?